rakelib/test.rake

```diff
@@ -1,33 +1,29 @@
+def run_rspec(*args)
+  require "logstash/environment"
+  LogStash::Environment.set_gem_paths!
+  require "rspec/core/runner"
+  require "rspec"
+  RSpec::Core::Runner.run([*args])
+end
```

Comment:
well... al least it's now DRY & isolated in a method, we can add conditions for Windoze if there is specific issues...